snuba/clickhouse/native.py

```diff
@@ -103,14 +130,42 @@ def execute(
                         types_check=types_check,
                         columnar=columnar,
                     )
+                    result_data: Sequence[Any]
+                    trace_output = ""
+                    if capture_trace:
+                        with capture_logging() as buffer:
+                            if settings:
+                                settings = {**settings, "send_logs_level": "trace"}
+                            else:
+                                settings = {"send_logs_level": "trace"}
+                            result_data = query_execute()
+                            # In order to avoid exposing PII the results are discarded
+                            result_data = [[], []] if with_column_types else []
+                            trace_output = buffer.getvalue()
+                    else:
+                        result_data = query_execute()
+
+                    profile_data: ClickhouseProfile = {
```

Comment:
so this is always available whether we capture the trace or not?

Reply:
Yes because I want to use tracing data for attribution on all queries.